### PR TITLE
Link logo in web setup wizard to back-end base URL

### PR DESCRIPTION
**Pull request by arnoudhgz**
### Description
When you get to the web setup wizard from the Magento back-end it was not possible to get back to the back-end from this page.

This commit makes the logo clickable and when you click it you will return the Magento back-end.

### Fixed Issues (if relevant)
1. magento/magento2#13948: 
Sidebar shortcut to admin dashboard (Magento logo on top left) has no link in web setup wizard
2. ...

### Manual testing scenarios
1. Admin dashboard > System > Web Setup Wizard
2. Click Magento logo on top left
3. You get back to admin dashboard

### Contribution checklist
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [x] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
Hi @arnoudhgz. Thank you for your contribution 
Here is some useful tips how you can test your changes using Magento test environment.
Add the comment under your pull request to deploy test or vanilla Magento instance:
- `@magento-engcom-team give me test instance` - deploy test instance based on PR changes
- `@magento-engcom-team give me {$VERSION} instance` - deploy vanilla Magento instance

For more details, please, review the [Magento Contributor Assistant](https://github.com/magento/magento2/wiki/Magento-Contributor-Assistant) documentation

File: setup/view/magento/setup/navigation/side-menu.phtml [hunk 1]
```diff
@@ -20,13 +27,13 @@
     ng-show="<?= implode( '&&', $expressions) ?>"
     >
 <nav class="admin__menu" ng-controller="mainController">
-    <span
-        class="logo logo-static"
```
Comment (mzeis) [inline setup/view/magento/setup/navigation/side-menu.phtml:24]: Hi @arnoudhgz, thanks for your contribution!
When you remove the CSSclass `logo-static` from the template, it should be possible to remove its declaration in the LESS file. Please can you look into this?

Thanks!

**Review by mzeis — CHANGES_REQUESTED** [review]

File: setup/view/magento/setup/navigation/side-menu.phtml (hunk 1 above)
Comment (arnoudhgz) [reply setup/view/magento/setup/navigation/side-menu.phtml:24]: @mzeis thank you for the feedback, this was indeed possible. 

It was also set in the `setup/pub/styles/setup.css` file (which was actually loaded on the page).

I removed it from both locations.

image

**Comment by magento-engcom-team** [comment]
Hi @arnoudhgz. Thank you for your contribution. 
We will aim to release these changes as part of 2.2.7. 
Please check the release notes for final confirmation.

Please, consider to port this solution to 2.3 release line.
You may use [Porting tool](https://github.com/magento/magento2/wiki/Simple-way-to-port-your-solutions-automatically) to port commits automatically.

**Pull request merged**